apps/roam/src/utils/conceptConversion.ts

```diff
@@ -0,0 +1,184 @@
+import { DiscourseNode } from "./getDiscourseNodes";
+import getDiscourseRelations from "./getDiscourseRelations";
+import type { DiscourseRelation } from "./getDiscourseRelations";
+import { Database } from "@repo/database/types.gen";
+import type { SupabaseContext } from "~/utils/supabaseContext";
+
+// When it's merged: import type { LocalConceptDataInput } from '@repo/database/input_types';
```

Comment:
When what is merged?

Reply:
It referred to the input_types file. I thought it was obvious. It's in ENG-415, which I guess I should have specified.